package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@paraswap/dex-lib",
-  "version": "2.9.53",
+  "version": "2.9.51-pancake-v2-mainnet.1",
```

Comment:
Shouldn't the version be "2.9.53-pancake-v2-mainnet.1"? (2.9.53 instead of 2.9.51)